floris/simulation/turbine.py

```diff
@@ -162,7 +281,11 @@ def power(
 def Ct(
     velocities: NDArrayFloat,
     yaw_angle: NDArrayFloat,
+    tilt_angle: NDArrayFloat,
```

Comment:
Need to add these arguments to the docstring below:
tilt_angle
ref_tilt_cp_ct
tilt_interp
correct_cp_ct_for_tilt